fuzz/fuzz_targets/bat_stdin_input.rs

```diff
@@ -5,15 +5,69 @@ use libfuzzer_sys::fuzz_target;
 /// This prints embedded bytes with a custom header and then reads from STDIN.
```

Comment:
Sorry to be pedantic about this, but I would say if you are going to simply reuse the existing target, just submit a Mayhemfile, Dockerfile, and workflow file (all three seem to be missing from this PR). On the other hand, if you are going to update the target, why not just create a new one as a copy? A large percentage of submitted targets never end up fuzzing the most recent source because somewhere along the line a breaking commit was made, so I'm being picky here to make sure that happens as infrequently as possible.